implementations/swift/Makefile

```diff
@@ -1,21 +1,53 @@
-build: ockam_app_lib
-	xcodebuild archive -project ockam/ockam_app/Ockam.xcodeproj/ -scheme "Portals, by Ockam" -configuration Debug -archivePath build/Ockam.xcarchive
-	@echo "Build complete.  See build/Ockam.xcarchive for the built app."
-	@echo "To execute run: build/Ockam.xcarchive/Products/Applications/Portals,\ by\ Ockam.app/Contents/MacOS/Portals,\ by\ Ockam"
-build_release: ockam_app_lib_release xcode_build
-xcode_build:
-	xcodebuild archive -project ockam/ockam_app/Ockam.xcodeproj/ -scheme "Portals, by Ockam" -configuration Release -archivePath build/Ockam.xcarchive
+ROOT_DIR := $(realpath $(dir $(firstword $(MAKEFILE_LIST)))/../..)
+
+# delete the swift build artifacts
+clean:
+	rm -rf "build/"
+
+# delete the swift build artifacts and the ockam command + ockam_app_lib artifacts
+very_clean: clean
+	$(MAKE) -f ../rust/Makefile clean
+
+# build the ockam command line application + ockam_app_lib crate in Debug mode
+build_ockam:
+	$(MAKE) -f ../rust/Makefile build
+
+# build the ockam command line application + ockam_app_lib crate in Release mode
+build_ockam_release:
+	$(MAKE) -f ../rust/Makefile build_release
+
+# build the Portals application in Debug mode
+build: build_ockam
+	# the xcode project expects the ockam executable to be found in the release directory
+	cp $(ROOT_DIR)/target/debug/ockam $(ROOT_DIR)/target/debug/release
+	make xcode_build_Debug
+
+# build the Portals application in Release mode
+build_release: build_ockam_release
+	make xcode_build_Release
+
+# build the Portals application in Release mode and sign it
+build_signed: build_ockam_release
+	make xcode_build_signed
+
+# build the Swift code in Debug or Release mode
+xcode_build_%:
```

Comment:
are you sure the wildcard won't intercept `xcode_build_signed`?

Reply:
Yes I tested it.